### PR TITLE
Dinner recipe without header and footer

### PR DESCRIPTION
**Pull request by mixrecovery19**
First Recipe - Dinner.

### CODE REVIEW

File: .hintrc
Comment (Fred-Khan) [inline .hintrc:1]: Please remove this file from your commit (git rm --cached .hintrc) and add to .gitignore file in your project root folder.

File: index.html
Comment (Fred-Khan) [inline index.html:1]: This file was modified in a way that causes conflict with a previous merge. You will need to do a git pull to resolve this conflict. 

1. Do the git pull command within VS Code terminal while the project is open. 
2. You should get the option to Resolve in Merge Editor. 
3. Choose Accept Incoming and Complete Merge for this file conflict. 
4. Once conflicts are resolved, commit and push.

File: recipe-chicken-with-apricot.jpg
Comment (Fred-Khan) [inline recipe-chicken-with-apricot.jpg:1]: All image assets should be in the images folder.

**Review by Fred-Khan — CHANGES_REQUESTED** [review]
Please amend changes as requested to gitignore, images and resolve the conflict with index.html.

File: recipe-chicken-with-apricot.jpg
Comment (mixrecovery19) [reply recipe-chicken-with-apricot.jpg:1]: Moved image to images file

File: index.html
Comment (mixrecovery19) [reply index.html:1]: Did git pull tells me I am already up to date.

File: .hintrc
Comment (mixrecovery19) [reply .hintrc:1]: added hintrc to .gitignore, then somehow deleted it.

**Review by Fred-Khan — APPROVED** [review]
Well done, Michael! Looking forward to seeing the breakfast and lunch recipes.

**Pull request merged**